src/addressUtilsShelley.c

```diff
@@ -504,6 +502,4 @@ void parseAddressParams(const uint8_t *wireDataBuffer, size_t wireDataSize, addr
 	default:
 		ASSERT(false);
 	}
-
-	VALIDATE(view_remainingSize(&view) == 0, ERR_INVALID_DATA);
```

Comment:
why the check that the view was fully consumed is no longer enforced?

Reply:
It should never have been here in the first place; it's just that before now, address params were always at the end of wireData, so the view contents was consumed in every usecase. This is no longer true; and the caller should be responsible for determining if more data are to be read from the view or not.